Kitodo-Command/src/main/java/org/kitodo/command/Command.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.command;
+
+import org.kitodo.api.command.CommandInterface;
+import org.kitodo.api.command.CommandResult;
+
+public class Command implements CommandInterface {
+
+    @Override
+    public CommandResult runCommand(Integer id, String command) {
+        return null;
```

Comment:
I just realised, that RuntimeException is unchecked, so the signature can stay as it is and everything is fine. :)